### DISCUSSION TITLE
Displaying user messages

### DISCUSSION THREAD

**Discussion by simonhackler** [bromberry-games/Inkvestigations#148] [UI/UX]
@zivky

Users can buy singular messages and they have a daily amount of messages.
Right now we only display this in the chat window. 
We should show this somewhere else.
We could show this on:
navbar
profile dropdown
edit page

**Comment by zivky** [comment]
We could make it part of the profile image.
like this
![image](https://github.com/bromberry-games/Inkvestigations/assets/79329549/3777dd17-72e2-48c3-8909-2a6f3ad4cf18)

**Reply by simonhackler** [reply]
Looks great!
But would need something like 50|25
Since we have singular messages that can be bought and subscription messages that refill daily.
Or should we only show the singular messages here?

**Reply by zivky** [reply]
Would it be crazy if users could see their total messages at a glance? What would be the advantage of separating which are the daily messages and which are the singular ones?

**Reply by simonhackler** [reply]
We are always using up daily messages first and then afterwards the singular ones.
We could combine it ofc. But I think it is relevant to know the amount of daily messages your have(or have left) and then to know the amount of singular messages.

**Reply by simonhackler** [reply]
I really like this now. This also means we can get rid of displaying the user messages inside of the chat and just have it next to the icon
